neutron/templates/agents/l3-agent.yaml.j2

```diff
@@ -30,12 +30,18 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: COMMAND
-              value: "neutron-l3-agent --config-file /etc/neutron/neutron.conf --config-file /etc/neutron/l3-agent.ini --config-file /etc/neutron/plugins/ml2/ml2-conf.ini"
+              value: "bash /tmp/l3-agent.sh"
```

Comment:
We also need to add ovs dependency here:

            - name: DEPENDENCY_CONTAINER
              value: "openvswitch-db-server,openvswitch-vswitchd"

So we make sure that ovs is running before issuing ovs-vsctl commands.